shared/teams/team/container.js

```diff
@@ -1,94 +1,67 @@
 // @flow
-import * as Constants from '../../constants/teams'
-import * as I from 'immutable'
 import * as React from 'react'
 import * as TeamsGen from '../../actions/teams-gen'
-import * as KBFSGen from '../../actions/kbfs-gen'
-import * as Chat2Gen from '../../actions/chat2-gen'
-import Team, {CustomComponent, type Props} from '.'
+import Team from '.'
+import CustomTitle from './custom-title/container'
 import {HeaderHoc} from '../../common-adapters'
-import {branch, connect, lifecycle, compose, type TypedState} from '../../util/container'
-import {navigateAppend} from '../../actions/route-tree'
-import {anyWaiting} from '../../constants/waiting'
-import {teamsTab} from '../../constants/tabs'
-
-import {membersListItemsConnector} from './members/container'
```

Comment:
before these things had their own connectors to pull this stuff out but really its likely simpler to reason about if they're just pure helpers. Now we have mapStateHelper and getRows which acts sorts like the old style connectors.